conf/cassandra.yaml

```diff
@@ -1583,7 +1583,7 @@ drop_compact_storage_enabled: false
 #         abort_threshold_kb: 0
 
 # Whether guardrails are enabled or not. Guardrails are disabled by default.
-# guardrails_enabled: true
+# guardrails_enabled: false
```

Comment:
I've just realized that this should be false by default. Doesn't make a big difference in practice since it's commented and `Config.guardrails_enabled` is already false.